pytest_flakefinder.py

```diff
@@ -72,7 +73,7 @@ def pytest_collection_modifyitems(self, items):
         # Also we want to @tryfirst so that we go before randomizing the list.
         for item in list(items):
             if not getattr(item.function, '_pytest_duplicated', None):
-                items.extend([item] * (self.flake_runs - 1))
+                items.extend([copy.copy(item)] * (self.flake_runs - 1))
```

Comment:
Doesn't that stick the same copy of `item` multiple times.  You might want a 
`copy.deepcopy([item] * (self.flake_runs - 1)` instead?

Reply:
sadly, you can't deepcopy item because it has some stuff inside that doesn't support it

Reply:
good catch though - I will fix it up.